mass-stop-start-hana-cloud/README.md

```diff
@@ -0,0 +1,65 @@
+
+To collect all databases, we utilize the *ListServiceInstances* command which is part of the *SAP Service Manager* catalog. The example assumes that all HANA Cloud service instances are labeled with the *service: hana-cloud*. This is necessary to identity them among other service instances that you might have in your spaces. This logic can easily be changed by modifying the *ListHanaCloudInstances* step in the example commands. Please also check [this documentation page](https://help.sap.com/docs/service-manager/sap-service-manager/filtering-parameters-and-operators) for more information on how to use the filter queries.
+
+![List Parameters](assets/list-parameters.png)
+
+:warning: The commands in this example might cause downtime for a large amount of databases. Please make sure that you are not using them on productive systems.
+
+## Requirements
+
+To use this example you'll need the following:
+
+* One or more HANA Cloud databases in the Cloud Foundry environment
+* Instance of [SAP Service Manager](https://help.sap.com/docs/service-manager/sap-service-manager/sap-service-manager) with plan *subaccount-admin* and a service key/bindin with default configurations.
+* Platform user with *Space Developer* role in the spaces where the databases reside
+
+:warning: If you are planning to use SAP Universal ID, please keep [SAP Note 3085908](https://launchpad.support.sap.com/#/notes/3085908) in mind
+
+Check out the following resources for more information:
+
+* [Deploy SAP HANA Cloud](https://developers.sap.com/tutorials/hana-cloud-deploying.html)
+* [Create Users and Manage Roles and Privileges](https://developers.sap.com/tutorials/hana-cloud-mission-trial-4.html)
+* [Creating Service Instances in Other Environments](https://help.sap.com/docs/service-manager/sap-service-manager/creating-instances-in-other-environments)
+* [Creating Service Bindings in Other Environments](https://help.sap.com/docs/service-manager/sap-service-manager/creating-service-bindings-in-other-environments)
+
+## How to use
+
+Import the content of [examples catalog](catalog.json) in your Automation Pilot tenant.
+The examples includes two commands - *MassStopHanaCloudInstances* and *MassStartHanaCloudInstances*. Navigate to any of them and to trigger them.
+
+You'll need to provide values for the following input keys:
+
+* *serviceKey* - Service Key for SAP Service Manager
+* *user* - Email or ID of your technical user
+* *password* - Password of your technical user
+* *identityProvider* - Optional: origin key of your identity provider. Defaults to sap.ids
+
+As mentioned before, Automation Pilot allows executions to be automatically triggered on regular intervals - hourly, daily, weekly, monthly or yearly. We can create two [Scheduled Executions](https://help.sap.com/docs/AUTOMATION_PILOT/de3900c419f5492a8802274c17e07049/96863a2380d24ba4bab0145bbd78e411.html) - one to stop all databases after working hours and another to start them up again on the next morning.
```

Comment:
We also can add to be taken in consideration that the supported time-zone it is UTC.